core/src/test/java/tech/pegasys/eth2signer/core/multikey/BlsArtifactSignerFactoryTest.java

```diff
@@ -94,7 +94,7 @@ void createsArtifactSignerFromKeyStoreUsingAbsolutePaths() {
   void nonExistentKeyStoreThrowsError() {
     final Path nonExistingKeystoreFile = configDir.resolve("someNonExistingKeystore");
     final FileKeyStoreMetadata fileKeyStoreMetadata =
-        new FileKeyStoreMetadata(nonExistingKeystoreFile, passwordFile);
+        new FileKeyStoreMetadata(nonExistingKeystoreFile, passwordFile, null);
```

Comment:
null seems to bad here, why can't we use Type.BLS to explicitly state we are dealing with BLS type?

Reply:
done